tingbot/error.py

```diff
@@ -43,11 +43,7 @@ def sad_tingbot_image():
 
 
 def error_screen(exc_info):
-    surface = pygame.display.get_surface()
-    if not surface:
-        return
-
-    screen = graphics.Surface(surface)
+    screen = graphics.screen
 
```

Comment:
Sorry, forgot to push! It's there now (05fc731af8273f2691daf13299f475f34ad57e1c).

> ...imagine having a non-visible app that just responds to button presses...

Good point! In this case, the simulator window could be lazy-loaded and called from either `create_main_surface` or `register_button_callback`. Then it'll only be created if it's needed.